doc/SdA.txt

```diff
@@ -177,14 +177,14 @@ The user can run the code by calling:
   python code/SdA.py
 
 By default the code runs 15 pre-training epochs for each layer, with a batch
-size of 1. The corruption levels are 0.1 for the first layer, 0.2 for the second,
+size of 20. The corruption levels are 0.1 for the first layer, 0.2 for the second,
 and 0.3 for the third. The pretraining learning rate is 0.001 and 
-the finetuning learning rate is 0.1. Pre-training takes 585.01 minutes, with 
-an average of 13 minutes per epoch. Fine-tuning is completed after 36 epochs
-in 444.2 minutes, with an average of 12.34 minutes per epoch. The final 
-validation score is 1.39% with a testing score of 1.3%. 
-These results were obtained on a machine with an Intel
-Xeon E5430 @ 2.66GHz CPU, with a single-threaded GotoBLAS.
+the finetuning learning rate is 0.1. Pre-training takes 71.43 minutes, with
+an average of 1.6 minutes per epoch. Fine-tuning is completed after 243 epochs
+in 441.3 minutes, with an average of 1.9 minutes per epoch. The final
+validation score is 1.7% with a testing score of 1.6%.
```

Comment:
People stopped using pre-processing as we don't know when we need to stop
the pre-processing. So this make using this much more complicated to use as
we have seen here. Do you want to try to find a case that give better
result? It could that that the pre-processing must be done for more or less
epoch. Maybe also lower batch_size could help have faster speed and keep
the good results.

On Thu, Feb 18, 2016 at 11:30 AM, Benjamin Irving <notifications@github.com>
wrote:

> In doc/SdA.txt
> <https://github.com/lisa-lab/DeepLearningTutorials/pull/142#discussion_r53339918>
> :
>
> >  and 0.3 for the third. The pretraining learning rate is 0.001 and
> > -the finetuning learning rate is 0.1. Pre-training takes 585.01 minutes, with
> > -an average of 13 minutes per epoch. Fine-tuning is completed after 36 epochs
> > -in 444.2 minutes, with an average of 12.34 minutes per epoch. The final
> > -validation score is 1.39% with a testing score of 1.3%.
> > -These results were obtained on a machine with an Intel
> > -Xeon E5430 @ 2.66GHz CPU, with a single-threaded GotoBLAS.
> > +the finetuning learning rate is 0.1. Pre-training takes 71.43 minutes, with
> > +an average of 1.6 minutes per epoch. Fine-tuning is completed after 243 epochs
> > +in 441.3 minutes, with an average of 1.9 minutes per epoch. The final
> > +validation score is 1.7% with a testing score of 1.6%.
>
> It does sound more tricky than just increasing the batch size as I did
> here. But it is a pity the speed is hit by having batch_size=1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lisa-lab/DeepLearningTutorials/pull/142/files#r53339918>
> .
>